### PR TITLE
4750 action bar messages

### PR DESCRIPTION
**Pull request by latin-panda**
# Description

Work in progress..

medic/cht-core#4750

# Code review items

- Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- Tested: Unit and/or e2e where appropriate
- Internationalised: All user facing text
- Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by latin-panda** [comment]
Hi @dianabarsan Could you please take a look? This one is small :) thanks!

File: webapp/package.json [hunk 1]
```diff
@@ -22,7 +22,6 @@
     "@angular/router": "^10.0.14",
     "@ngrx/effects": "^10.0.0",
     "@ngrx/store": "^10.0.0",
-    "@ngrx/effects": "^10.0.0",
```
Comment (dianabarsan) [inline webapp/package.json:25]: Is this a merge error? I think we still need effects :)

**Review by dianabarsan — APPROVED** [review]
So awesome! Just one tiny comment inline! Feel free to merge after that's fixed!

File: webapp/package.json (hunk 1 above)
Comment (dianabarsan) [reply webapp/package.json:25]: Ah sorry, later I saw it was added twice!

**Comment by dianabarsan** [comment]
I'm going to merge this, as I need to add actionbars for reprorts as well!

**Pull request merged**